app/client/src/features/events/DeleteEvent.tsx

```diff
@@ -0,0 +1,18 @@
+import { graphql } from 'react-relay';
+
+const DELETE_EVENT_MUTATION = graphql`
+    mutation DeleteEventMutation($input: DeleteEvent!) {
+        deleteEvent(event: $input) {
+            isError
+            message
+            body {
+                id 
+                title
+                topic
+                startDateTime
+            }
+        }
+    }
+`;
+
+export { DELETE_EVENT_MUTATION }
```

Comment:
I put it in a different file to keep it consistent with how we have CreateEvent and Update Event outside of where they are actually used, and then I use the DeleteEventMutation in the Delete Event Settings. Would it be better if I put this file in the same folder as DeleteEventSettings?

Reply:
Ah, I see.  CreateEvent and UpdateEvent are both forms.  However, DeleteEvent is not a form.  I think it'd be best here to create a simple react component in this file that just renders a button that reads "Delete Event" or something like that (along with a modal, like "are you sure").  Then, you'd import this button inside of EventSettings.  I think this may eliminate the need `DeleteEventSettings`.